libs/blocks/quiz/quiz.css

```diff
@@ -1,980 +1,384 @@
-.quiz { 
-  min-height: 1000px;
-  display: flex;
-  align-items: center;
-  flex-direction: column;
+.quiz-page * {
+  box-sizing: border-box;
 }
 
-.quiz > div:not([class]) {
-  display:none;
+.quiz-page .feds-topnav-wrapper {
+  padding: 0 20px;
 }
 
-.quiz-container {
-  display: flex;
-  align-items: center;
-  justify-content: flex-start;
-  flex-direction: column;
-  color: var(--color-white);
-  width: 100%;
+div[data-valign="middle"] {
+  align-self: center;
 }
 
-.quiz-container .foreground {
-  position: relative;
-  margin: var(--spacing-m) 0 var(--spacing-s);
-  display: flex;
-  flex-direction: column;
-  gap: var(--spacing-m);
-  justify-content: center;
+.quiz-load {
   text-align: center;
 }
 
-.quiz-option {
-  display: flex;
-  align-items: stretch;
-  flex: 1;
-  flex-direction: row;
-  width: auto;
-  margin: 5rem 0;
-  justify-content: center;
-  overflow-y: auto;
-}
-
-.quiz-container.small .foreground {
-  padding: var(--spacing-xl) 0;
-}
-
-.quiz-container.large .foreground {
-  padding: var(--spacing-m) 0;
-  gap: 0;
-}
-
-.quiz-container.small,
-.quiz-container.quiet,
-.quiz-container.inline {
-  min-height: 360px;
-}
-
-.quiz-container.inline hr {
-  margin-top: var(--spacing-m);
-  margin-bottom: 0;
-}
-
-.quiz-container.light,
-.quiz-container.quiet,
-.quiz-container.inline {
-  color: var(--text-color);
+.quiz {
+  min-height: calc(100vh - 131px);
```

Comment:
`131px` sounds a bit arbitrary. Could you add a comment or replace the value with a variable that has an appropriate name?

Reply:
I'll add a comment, but this is basically 100vh - gnav and gfooter heights to ensure that we avoid odd whitespace below gfooter. (see the before page with a tall browser window to see what I'm talking about here).

Reply:
It does work right now, but it might not be an ideal solution long-term. Once we have site-wide alerts on top of the navigation, this calculation will fail. Footers for Korea and China can span multiple rows (even the limited footer present on this page) due to legal requirements, so this approach will not yield expected results.

This shouldn't be blocker for the purposes of this CSS refactoring PR, but you might want to open a follow-up/technical debt story to investigate a better approach to this.

Reply:
Thanks @overmyheadandbody. I create this story for later: https://jira.corp.adobe.com/browse/MWPW-136135